Description: Allow build without qml-box2d submodule
 gcompris-qt (0.50-2) unstable; urgency=medium
 .
   * The "not yet 1500-lines debian/copyright" release.
   * Added src/activities/instruments/resource/drum_kit.ogg and
     src/activities/details/details.svg to debian/copyright (spotted by
     Thorsten Alteholz).  Plus a number of PNG and SVG files containing a
     public domain notice.
   * Include upstream patch to support building without qml-box2d, which
     should be separately packaged before we can properly use it.
Author: Yann Dirson <dirson@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- gcompris-qt-0.50.orig/CMakeLists.txt
+++ gcompris-qt-0.50/CMakeLists.txt
@@ -276,6 +276,8 @@ include(qt_helper)
 getQtQmlPath(_qt_qml_system_path)
 set (_box2d_system_dir "${_qt_qml_system_path}/Box2D.2.0")
 
+if(NOT QML_BOX2D_MODULE STREQUAL disabled)
+
 if (${QML_BOX2D_MODULE} STREQUAL "submodule")
   message(STATUS "Building qml-box2d module from submodule")
   set(_need_box2d_submodule "TRUE")
@@ -338,3 +340,5 @@ if (_need_box2d_submodule)
     install(DIRECTORY ${_box2d_install_dir} DESTINATION lib/qml)
   endif()
 endif()
+
+endif()
